update-dependencies/CliDependencyHelper.cs

```diff
@@ -35,11 +49,12 @@ public static string GetSharedFrameworkVersion(string cliVersion)
             return sharedFrameworkVersion;
         }
 
-        private static string GetCommitHash(string cliVersion)
+        private string GetCommitHash()
         {
-            using (ZipArchive archive = DownloadCliInstaller(cliVersion).Result)
+            using (ZipArchive archive = DownloadCliInstaller().Result)
             {
-                ZipArchiveEntry versionTxtEntry = archive.GetEntry($"sdk\\{cliVersion}\\.version");
+                string zipEntryName = CliMajorVersion == 1 ? $"sdk/{CliVersion}/.version" : $"sdk\\{CliVersion}\\.version";
```

Comment:
I think you should be able to use "sdk/{CliVersion}/.version" in all cases and ZipArchive will handle it (this is the behavior I saw on the symbol signing stuff at least).